portfolio/src/main/webapp/script.js

```diff
@@ -15,9 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
+function addRandomGreeting() 
```

Comment:
This function no longer operates on greetings, so update the function name. Same comment applies to the rest of this file.
Left this comment as part of a previous review; is this code the most up-to-date?